docs/dev/connectors/elasticsearch.md

```diff
@@ -198,10 +198,6 @@ List<HttpHost> httpHost = new ArrayList<>();
 httpHosts.add(new HttpHost("127.0.0.1", 9200, "http"));
 httpHosts.add(new HttpHost("10.2.3.1", 9200, "http"));
 
-Map<String, String> bulkRequestsConfig = new HashMap<>();
```

Comment:
Remove `Map` from the imports.

Reply:
It is still used when processing requests.
It can be removed from the Scala code snippet, though.